test/Makefile

```diff
@@ -18,7 +18,7 @@ endif
 $(TESTS):
 	@cd $(SRCDIR) && \
 	$(ULIMIT_TEST) \
-	$(call PRINT_JULIA, $(call spawn,$(JULIA_EXECUTABLE)) --check-bounds=yes --startup-file=no ./runtests.jl $@)
```

Comment:
Can we make sure we still have this file? It's very useful to run the tests without going through makefile in order to specify different options.